src/dashboard/src/main/migrations/0053_quote_args.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def data_migration_put_quotes_in_compress_aip_args(apps, schema_editor):
+    StandardTaskConfig = apps.get_model('main', 'StandardTaskConfig')
+
+    tasks = (
+        # upload-archivesspace.py
+        (
+            '10a0f352-aeb7-4c13-8e9e-e81bda9bca29',
+            u'--host="%host%" --port="%port%" --user="%user%"'
+            u' --passwd="%passwd%" --dip_location="%SIPDirectory%"'
+            u' --dip_name="%SIPName%" --dip_uuid="%SIPUUID%"'
+            u' --restrictions="%restrictions%" --object_type="%object_type%"'
+            u' --xlink_actuate="%xlink_actuate%" --xlink_show="%xlink_show%"'
+            u' --use_statement "%use_statement%" --uri_prefix "%uri_prefix%"'
```

Comment:
Why are you changing some of the optional arguments of the form `--x n` to the form `--x=n` but not others? E.g., `--use_statement "%use_statement%"` lacks the `=`. Is there a reason?